src/Euclidean.jl

```diff
@@ -19,6 +19,10 @@ struct Euclidean{T<:Tuple} <: Manifold where {T} end
 Euclidean(n::Int) = Euclidean{Tuple{n}}()
 Euclidean(m::Int, n::Int) = Euclidean{Tuple{m,n}}()
 
+function representation_size(::Euclidean{S}, ::Type{T}) where {S,T<:Union{MPoint, TVector, CoTVector}}
```

Comment:
`T` should also permit `Array`s here.

Reply:
Not really, `T` is supposed to refer what *kind* of object is represented (point on the manifold, some element of a vector space like tangent vector, cotangent vector, Riemann tensor etc.). I don't really like this solution, it's not clear what it is, but I thought some functionality of this sort is necessary and this was my first idea.